gpcontrib/arenadata_toolkit/expected/adb_collect_table_stats_test.out

```diff
@@ -46,20 +49,6 @@ PARTITION BY RANGE(b)
 	PARTITION pub_db_files_history_t1 START ('10'::bigint) END ('1000000'::bigint),
 	PARTITION pub_db_files_history_t2 START ('1000001'::bigint) END ('2000000'::bigint)
 );
--- There are not "db_files_history" and partitions
```

Comment:
Why was this check removed?

Reply:
There is not any reason to check non-existed relation, because I added cleaning up before each test starts.